src/types/redis_zset.cc

```diff
@@ -557,7 +558,7 @@ rocksdb::Status ZSet::Remove(const Slice &user_key, const std::vector<Slice> &me
   return storage_->Write(storage_->DefaultWriteOptions(), batch->GetWriteBatch());
 }
 
-rocksdb::Status ZSet::RemoveRangeByScore(const Slice &user_key, ZRangeSpec spec, int *ret) {
+rocksdb::Status ZSet::RemoveRangeByScore(const Slice &user_key, CommonRangeScoreSpec &spec, int *ret) {
```

Comment:
```c++
rocksdb::Status ZSet::RemoveRangeByScore(const Slice &user_key, const CommonRangeScoreSpec &spec, int *ret) {
```

?

Reply:
Good to have. Perhaps we focus on the functionality in this PR, and at least the style is regression or quite unacceptable, I can gather them to do in a follow-up.

Reply:
After a closer look, it writes:

```
spec.removed = true;
```

so that we cannot use `const` here. But it comes to me a smell we can investigate further later.

Reply:
i guess we can just pass-by-value. I guess copying it is lightweight :)